lib/purgatory/purgatory.rb

```diff
@@ -44,9 +44,14 @@ def soul_with_changes
   end
 
   def approve!(approver = nil)
-    return false if approved? 
-    if soul_with_changes.save
-      soul.send(performable_method[:method],*performable_method[:args]) if performable_method
+    return false if approved?
+    success = nil
```

Comment:
@dubroe  Yeah, so bump the version number... maybe `3.1` then? I think it's okay to do that and note it in the docs.